rpm2cpio.yaml

```diff
@@ -0,0 +1,41 @@
+package:
+  name: rpm2cpio
+  version: 14.2.0
+  epoch: 0
+  description: Convert .rpm files to cpio format
+  copyright:
+    - license: BSD-2-Clause
+  dependencies:
+    runtime:
+      - libarchive-tools
+
+environment:
+  contents:
+    packages:
+      - busybox
+
+pipeline:
+  - uses: fetch
+    with:
+      uri: https://raw.githubusercontent.com/freebsd/freebsd-ports/refs/heads/main/archivers/rpm2cpio/files/rpm2cpio
+      expected-sha256: 2841bacdadde2a9225ca387c52259d6007762815468f621253ebb537d6636a00
+      extract: false
+
+  - runs: |
+      install -Dm755 rpm2cpio ${{targets.destdir}}/usr/bin/rpm2cpio
+      sed -i 's/tar/bsdtar/g' ${{targets.destdir}}/usr/bin/rpm2cpio
+
+update:
+  enabled: false
+  exclude-reason: single script file which hasn't been updated in last 10 years.
+
+test:
+  environment:
+    contents:
+      packages:
+        - curl
+  pipeline:
+    - runs: |
+        rpm2cpio
+        curl -L -o yq.rpm https://kojipkgs.fedoraproject.org//packages/yq/4.43.1/5.fc40/src/yq-4.43.1-5.fc40.src.rpm
```

Comment:
We shouldn't fetch a pinned version of yq. Let's retrieve the latest release of cosign instead (that version may be pulled from koji in the future): https://github.com/sigstore/cosign/releaseshttps://github.com/sigstore/cosign/releases

Reply:
Thanks, done!